.github/workflows/deploy-docs.yml

```diff
@@ -10,30 +10,37 @@ on:
   workflow_dispatch:
 
 jobs:
-  execute-notebooks:
+  execute-and-deploy:
     runs-on: 1-gpu-runner
     if: github.repository == 'sgl-project/sglang'
+    defaults:
+      run:
+        working-directory: docs
     steps:
       - name: Checkout code
         uses: actions/checkout@v3
+        with:
+          path: .
 
       - name: Set up Python
         uses: actions/setup-python@v4
         with:
           python-version: '3.9'
 
       - name: Install dependencies
+        working-directory: .
```

Comment:
remove this and update `pip install -r docs/requirements.txt` to `pip install -r requirements.txt`